src/solvers/dg_tree/dg.jl

```diff
@@ -268,19 +333,25 @@ include("basis_lobatto_legendre.jl")
           surface_flux=flux_central,
           volume_integral::AbstractVolumeIntegral=VolumeIntegralWeakForm(),
           mortar=MortarL2(basis))
+    DGSEM(; RealT=Float64, polydeg::Integer,
+            surface_flux=flux_central,
+            volume_integral::AbstractVolumeIntegral=VolumeIntegralWeakForm(),
+            mortar=MortarL2(basis))
```

Comment:
This docstring is redundant with the lines above?

Reply:
Also, shouldnt it be
```suggestion
    DGSEM(; RealT=Float64, polydeg::Integer,
            surface_integral=SurfaceIntegralWeakForm(),
            volume_integral::AbstractVolumeIntegral=VolumeIntegralWeakForm(),
            mortar=MortarL2(basis))
```
for consistency? Also, why the dispatch on `AbstractVolumeIntegral`?

Reply:
I removed the restriction to `AbstractVolumeIntegral` and the other docstring in 05801bfa.
DGSEM uses the `SurfaceIntegralWeakForm` by definition. We could allow more general surface terms, but we don't have a good reason to do so right now. We need to keep the keyword argument `surface_flux` for backward compatibility.

Reply:
Besides the weak/strong argument, an additional reason I see for the abstraction with the SurfaceIntegral type is to pave the way for alternative surface terms such as SAT (simultaneous approximation terms) that sometimes do not follow our "numerical flux" structure.

Reply:
Yeah, that's something I had in mind, too.